oci.go

```diff
@@ -245,8 +260,15 @@ func processCgroupsPathForResource(ociSpec oci.CompatOCISpec, resource string) (
 		}
 	}
 
-	if cgroupMountFound == false {
-		return "", fmt.Errorf("cgroupsPath is absolute, cgroup mount MUST exist")
+	if !cgroupMountFound {
+		if isPod {
+			return "", fmt.Errorf("cgroupsPath %q is absolute, cgroup mount MUST exist",
```

Comment:
Oh okay got it. But what's wrong about this wording ? If I add
```
"absolute cgroupsPath %q does not exist"
```
This is not really the problem here. Indeed, the OCI spec says that in case of absolute path, we don't want to check if this path exists, but we want to retrieve the mount cgroup which is supposed to be the prefix to concatenate to the cgroup path provided. In that case, we say that we got the following absolute path, but we cannot use it because cgroup mount path is empty.